template/template.go

```diff
@@ -27,3 +27,7 @@ func Evaluate(content []byte, variables Variables) ([]byte, error) {
 		return []byte(saveValue)
 	}), variableErrors
 }
+
+func EvaluateNil(content []byte) []byte {
+	return templateFormatRegex.ReplaceAll(content, []byte("nil"))
```

Comment:
does this replace all `{{foo}}`s with `nil`? May want to make those empty strings instead since that's the type those normally resolve to. Could result in validation errors otherwise.